From 5736ac0aa790c56eecf4e344e1ec25a5b4bc1d77 Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Wed, 19 Jun 2024 22:54:15 +0200
Subject: [PATCH] unbundle: add missing dav1d targets

fixes "ERROR Unresolved dependencies.
//third_party/crabbyavif:crabbyavif_dav1d_bindings(//build/toolchain/linux/unbundle:default)
  needs //third_party/dav1d:dav1d_config(//build/toolchain/linux/unbundle:default)
//third_party/crabbyavif:crabbyavif_dav1d_bindings(//build/toolchain/linux/unbundle:default)
  needs //third_party/dav1d:dav1d_headers(//build/toolchain/linux/unbundle:default)"
---
 build/linux/unbundle/dav1d.gn | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/build/linux/unbundle/dav1d.gn b/build/linux/unbundle/dav1d.gn
index 3d65158bc6144..a6d005a6622e3 100644
--- a/build/linux/unbundle/dav1d.gn
+++ b/build/linux/unbundle/dav1d.gn
@@ -21,3 +21,14 @@ source_set("dav1d") {
   deps = [ ":dav1d_shim" ]
   public_configs = [ ":system_dav1d" ]
 }
+
+group("dav1d_headers") {
+  public_deps = [ ":dav1d_shim" ]
+}
+
+config("dav1d_config") {
+  configs = [
+    ":system_dav1d",
+    ":dav1d_shim_config",
+  ]
+}
